telethon/tl/custom/button.py

```diff
@@ -72,7 +72,7 @@ def inline(text, data=None):
         if len(data) > 64:
             raise ValueError('Too many bytes for the data')
 
-        return types.KeyboardButtonCallback(text, data)
+        return types.KeyboardButtonCallback(text, data, requires_password)
```

Comment:
Please remove the changes in this file with the instructions given in https://github.com/LonamiWebs/Telethon/pull/1580#issuecomment-703153566.